dpti/workflows/prefect/prefect_task_hash.py

```
@@ -0,0 +1,81 @@
+import hashlib
+from functools import partial
+from typing import Any, Dict, Optional, Union
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)